content/en/quickstart/verification-cheat-sheet.md

```diff
@@ -0,0 +1,35 @@
+---
+type: docs
+category: Quickstart
+description: Verification Cheat Sheet
+title: OIDC Verification Cheat Sheet
+weight: 20
+---
+
+## Identity Verification Cheat Sheet
+
+### Verying identity from OIDC issuers
+
+To verify a signature created with an OIDC issuer, you need to know the following:
+
+* `certificate-identity` : the email address associated with the signer for the given OIDC issuer
```

Comment:
It may not always be an email address. The help text from cosign says "Valid values include email address, DNS names, IP addresses, and URIs."